bin/pbiviz-start.js

```diff
@@ -52,6 +56,13 @@ program.parse(options);
 let cwd = process.cwd();
 let server, builder;
 
+if (!CertificateTools.getCertFile(config, true)) {
+    CertificateTools.createCertFile(config);
```

Comment:
Will it ask developer to install a cert?

Reply:
No